tests/regression_test.py

```diff
@@ -0,0 +1,144 @@
+"""Tests for PROPKA 3.1"""
+import logging
+import os
+import re
+from pathlib import Path
+import pytest
+import pandas as pd
```

Comment:
If pandas is needed for the tests then the `setup.py` should contain 
```python
   install_requires=[],
   tests_require=[
        'pytest',
        'pandas', 
    ],   
```
as [arguments for `setup()`](https://setuptools.readthedocs.io/en/latest/setuptools.html#new-and-changed-setup-keywords).

(I don't remember that propka has any dependencies, otherwise they'd go in `install_requires`)

Reply:
I was trying to figure this out the other day and ended up following the guidance in https://packaging.python.org/discussions/install-requires-vs-requirements/.  Since the tests aren't needed to make PROPKA "minimally run correctly", I thought pandas should go in requirements.txt.

Reply:
Addressed in https://github.com/Electrostatics/propka-3.1/commit/ebddfa1f9fcec30f1e73ef8bd6b75696c7a35863 by adding the `tests_require` statement to setup.py.  Note that this functionality is deprecated so we should leave the dependencies in `requirements.txt` as well.